Library/ViewModels/SearchViewModelTests.swift

```diff
@@ -134,20 +134,20 @@ internal final class SearchViewModelTests: TestCase {
 
         XCTAssertEqual(
           self.dataLakeTrackingClient.events.last,
-          "Card Clicked"
+          "CTA Clicked"
         )
         XCTAssertEqual(
           self.segmentTrackingClient.events.last,
-          "Card Clicked"
+          "CTA Clicked"
         )
 
         let dataLakeProperties = self.dataLakeTrackingClient.properties.last
         let segmentProperties = self.segmentTrackingClient.properties.last
 
         XCTAssertEqual("search", dataLakeProperties?["context_page"] as? String)
-        XCTAssertEqual("project", dataLakeProperties?["context_type"] as? String)
+        XCTAssertEqual("results", dataLakeProperties?["context_type"] as? String)
         XCTAssertEqual("search", segmentProperties?["context_page"] as? String)
-        XCTAssertEqual("project", segmentProperties?["context_type"] as? String)
+        XCTAssertEqual("results", segmentProperties?["context_type"] as? String)
```

Comment:
Can you add tests for all the other properties defined for this event as well?

Reply:
The tests for the other properties are already taken care of in KSRAnalyticsTest, but I can always add it here if that's ok.

Reply:
Yeah add them in here just because we are checking page and type anyway.